com.github.rizinorg.Cutter.yml

```diff
@@ -0,0 +1,29 @@
+#flatpak-builder --force-clean --user --install ./build ./com.github.rizinorg.cutter.yml
+app-id: com.github.rizinorg.Cutter
```

Comment:
The app-id we use for Cutter is `re.rizin.cutter`.
Please check this file for reference: https://github.com/rizinorg/cutter/blob/master/src/re.rizin.cutter.appdata.xml

Reply:
I've updated it. Thanks for pointing it out